src/App.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
```

Comment:
Just to note, that convention is that React files are `.jsx` now, although `.js` does work too but its helpful to quickly distinguish between the types